### ISSUE TITLE
KERN-5788 breaks when /vmlinuz is missing

### ISSUE THREAD

**Issue by rhyven** [CISOfy/lynis#175]
On my Debian 8.3 systems with 4.3.3-grsec kernel, the symlink /vmlinuz doesn't exist.  This breaks the tests in KERN-5788.

I'm not sure if this caused by Debian 8.3, or if it's grsecurity that's doing it... I have no Debian systems without grsec installed.  I've experienced it on six separate VM's, all running Deb8.3+grsec.

@mboelen two questions:
 - Have you observed this elsewhere?
 - Do you think Lynis should identify a missing /vmlinux and code around it?

----

If you want to code around it, this is a rough snippet we could use:

```
if [ -f /vmlinuz ]; then 
    FINDKERNFILE=`readlink -f /vmlinuz`
    FINDKERNEL=`dpkg -S ${FINDKERNFILE} 2> /dev/null | awk -F : '{print $1}'`
else
    if [ -e /dev/grsec ]; then
        FINDKERNEL=linux-image-`uname -r`
    else
        ReportException
    fi
fi
```

Let me know how you'd like to approach  :-)

**Comment by mboelen** [comment]
Yes, we observed this. Seems your proposal might be a perfect fit. Will you create the patch?

**Comment by rhyven** [comment]
Fixed with PR #178

**Comment by mboelen** [comment]
Thanks, PR looks good. Made a few small changes to enhance the check (elif).
